City-of-Helsinki/drupal-helfi-platform-test
# Draft: PLAENH-42 - Gredi integration media

# PLAENH-42
Gredi Asset integration with media library
Module developed here https://github.com/City-of-Helsinki/drupal-module-helfi-gredi

## What was done
Installed the module drupal-module-helfi-gredi - that comes with:
- A new media type Gredi image with a new new media source
- A media library view that is injected in the media Library browser to allow browsing and picking images from the Gredi library. 
- Option to create a new media which uploads to a designated folder in Gredi

Configured liftup paragraph component image to allow selecting this new media type, to showcase the module functionality within an existing media field type

## How to install

* Make sure your instance is up and running on correct branch.
  * `git pull origin INSERT_BRANCH_HERE`
  * install from existing config or just import the config over existing one (if compatible with dev branch)

## How to test
- Add on a page or edit an existing page to have the liftup component
- Click on the media butotn to load the media library usual modal
- Select the tab Gredi image and you should see a list of images pulled down from the Gredi library
- Select an image from there and attach it to the page
- The image should be saved into a local media object and displayed in the conditions as any regular image media object.


## Designers review



## Other PRs
No
